README.md

```diff
@@ -33,10 +33,15 @@ Using Composer to manage standard sites on Pantheon not started with these upstr
 * `terminus composer my-site.dev -- update`
 
 ## Installation
-For help installing, see [Manage Plugins](https://pantheon.io/docs/terminus/plugins/)
+
+### Installing via Terminus 3
+`terminus self:plugin:install pantheon-systems/terminus-composer-plugin`
+
+On older versions of Terminus:
 ```
 mkdir -p ~/.terminus/plugins
-composer create-project --no-dev -d ~/.terminus/plugins pantheon-systems/terminus-composer-plugin:~1
+curl https://github.com/pantheon-systems/terminus-composer-plugin/archive/refs/tags/1.1.0.tar.gz -L | tar -C ~/.terminus/plugins -xvz
```

Comment:
Don't you think that `composer create-project` is better than curl-piped-to-tar?

Reply:
Good question, I chose that because the https://github.com/pantheon-systems/terminus-plugin-example repo uses the curl method and I operated under the assumption there was a reason for it.

Reply:
I think the older instructions simply were not updated. I honestly could go either way, if curl is better, but we should try to be consistent if we're going to make changes. I think that `create-project` is easier to read.